packages/components/buttons/preset/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@commercetools-uikit/buttons",
```

Comment:
Maybe we should be explicit in marking those packages as "presets". For example `@commercetools-uikit/preset-buttons`.